packages/core/remappings.txt

```diff
@@ -0,0 +1,7 @@
+@openzeppelin/=../../node_modules/@openzeppelin/
```

Comment:
I'm guessing there's no way to make these dynamic, right? It would be nice to use node to dynamically figure out where these go, since they can technically be at ../../node_modules OR ./node_modules. I wouldn't expect that to be a feature, though, since it's a pretty niche need.

Reply:
there is a way, actually. I just did not manage to get it working correctly: https://book.getfoundry.sh/config/hardhat#instructions

we can improve this later...this is meant to be an mvp.

Reply:
SG!